src/mixins/filing-template-mixin.ts

```diff
@@ -866,6 +867,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     // store Business Start Date
     this.setCorrectionStartDate(filing.changeOfRegistration.startDate || null)
 
+    // default this flag to true for change filing
+    this.setSpecialResolutionConfirmStateValidity(true)
```

Comment:
How about just initializing this flag to True in state-models.ts (like the other ReviewCertify flags) ?  If there is a special reason this flag needs to be initialized as False then it should be its component's responsibility to do that.

Also, please update the app version :)

Reply:
I think your way is better. This flag is only used in the `CreateSpecialResolutionSummary`, which sets it to false onload, and initialize it to true in the state model works.